pilot/pkg/security/authz/builder/builder.go

```diff
@@ -64,11 +64,11 @@ func (b Builder) BuildHTTP() []*httppb.HttpFilter {
 	var filters []*httppb.HttpFilter
 
 	if denyConfig := build(b.denyPolicies, b.trustDomainBundle,
-		false /* forTCP */, true /* forDeny */, b.isIstioVersionGE15); denyConfig != nil {
```

Comment:
the comment is actually referring to the argument name in the `build()` function definition.

Reply:
I got it, thank you !